ar.c

```diff
@@ -38,8 +38,12 @@ static int ar_version(lua_State *L) {
 }
 
 //////////////////////////////////////////////////////////////////////
+#if !defined LUA_VERSION_NUM
```

Comment:
Can this #if be refactored into a header (perhaps ar.h?) so we can avoid copying it into ar_entry.c?